technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -0,0 +1,493 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" initialize-with="." default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Historische Musikwissenschaft (note)</title>
```

Comment:
For styles fixed to a non-English locale, please include the language in the style title (i.e., `<title>Technische Universität Dresden - Historische Musikwissenschaft (note, German)</title>`)